backend/docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
       - .:/app:z
 
   db:
-    image: docker.io/library/postgres:12
+    image: docker.io/library/postgres:14
```

Comment:
Can't we go to 15?

Reply:
We're on 14 now on the production environment.

Reply:
Right, forgot, that this is not the real prod conf